addons/binding/org.openhab.binding.sleepiq/README.md

```diff
@@ -0,0 +1,92 @@
+
+### Chamber Channel Group
+
+All channels within this group are read-only.
+
+| Channel ID           | Item Type | Description                                                                                                     |
+|----------------------|-----------|-----------------------------------------------------------------------------------------------------------------|
+| inBed                | Switch    | The presence of a person or object on the chamber                                                               |
+| sleepNumber          | Number    | The Sleep Number setting of the chamber                                                                         |
+| pressure             | Number    | The current pressure inside the chamber                                                                         |
+| lastLink             | String    | The amount of time that has passed since a connection was made from the chamber to the cloud service (HH:MM:SS) |
+| alertId              | Number    | Identifier for an alert condition with the chamber                                                              |
+| alertDetailedMessage | String    | A detailed message describing an alert condition with the chamber                                               |
+
+## Items
+
+Here is a sample item configuration:
+
+```
+Switch      MasterBedroom_SleepIQ_InBed_Alice          "In Bed [%s]"        { channel="sleepiq:dualBed:1:master:left#inBed" }
+Number      MasterBedroom_SleepIQ_SleepNumber_Alice    "Sleep Number [%s]"  { channel="sleepiq:dualBed:1:master:left#sleepNumber" }
+Number      MasterBedroom_SleepIQ_Pressure_Alice       "Pressure [%s]"      { channel="sleepiq:dualBed:1:master:left#pressure" }
+String      MasterBedroom_SleepIQ_LastLink_Alice       "Last Update [%s]"   { channel="sleepiq:dualBed:1:master:left#lastLink" }
+Number      MasterBedroom_SleepIQ_AlertId_Alice        "Alert ID [%s]"      { channel="sleepiq:dualBed:1:master:left#alertId" }
+String      MasterBedroom_SleepIQ_AlertMessage_Alice   "Alert Message [%s]" { channel="sleepiq:dualBed:1:master:left#alertDetailedMessage" }
+
+Switch      MasterBedroom_SleepIQ_InBed_Bob            "In Bed [%s]"        { channel="sleepiq:dualBed:1:master:right#inBed" }
+Number      MasterBedroom_SleepIQ_SleepNumber_Bob      "Sleep Number [%s]"  { channel="sleepiq:dualBed:1:master:right#sleepNumber" }
+Number      MasterBedroom_SleepIQ_Pressure_Bob         "Pressure [%s]"      { channel="sleepiq:dualBed:1:master:right#pressure" }
+String      MasterBedroom_SleepIQ_LastLink_Bob         "Last Update [%s]"   { channel="sleepiq:dualBed:1:master:right#lastLink" }
+Number      MasterBedroom_SleepIQ_AlertId_Bob          "Alert ID [%s]"      { channel="sleepiq:dualBed:1:master:right#alertId" }
+String      MasterBedroom_SleepIQ_AlertMessage_Bob     "Alert Message [%s]" { channel="sleepiq:dualBed:1:master:right#alertDetailedMessage" }
+```
```

Comment:
I appreciate you saying so! I have benefited from well-documented bindings many times in the past.